snap/hooks/install

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -e
+
+# get the values of $SNAP_DATA and $SNAP using the current symlink instead of
+# the default behavior which has the revision hard-coded, which breaks after
+# a refresh
+SNAP_DATA_CURRENT=${SNAP_DATA/%$SNAP_REVISION/current}
+SNAP_CURRENT=${SNAP/%$SNAP_REVISION/current}
+
+# install all the config files from $SNAP/config/SERVICE/res/configuration.toml
+# into $SNAP_DATA/config and replace $SNAP_COMMON environment variable in the config files
+mkdir -p "$SNAP_DATA/config"
+if [ ! -f "$SNAP_DATA/config/app-service-configurable/res/configuration.toml" ]; then
```

Comment:
Any way to make this more dynamic so we don't have to update this file each time we add a profile?